examples/text-generation/run_generation.py

```diff
@@ -288,19 +303,49 @@ def generate():
         HabanaProfile.disable()
         # Compilation
         logger.info("Graph compilation...")
+        if len(args.simulate_dyn_prompt) > 0:
+            dyn_prompt_lens = [int(k) for k in args.simulate_dyn_prompt.split(",")]
+        else:
+            dyn_prompt_lens = None
```

Comment:
This wouldn't be needed with the suggestion above for defining the simulate_dyn_prompt arg.